recipe/0125-rendering-3d/index.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Including a 3D Rendering
+id: 93
+layout: recipe
+tags: [tbc]
+summary: "You want to include a 3D model representation"
+---
+
+
+## Use Case
+
+You want to view an object from any orbital point of view or zoom level.
+You want to view an object in augmented reality/XR in order to gain a better understanding of its dimensions and physical properties.
+
+## Implementation notes
+
+Add a [`rendering`](https://iiif.io/api/presentation/3.0/#rendering) to your manifest either at the root level, or any applicable child resource.
+The rendering MUST have a dereferenceable `id` linking to the supplied 3D model, a `type` of `Model`, and a `label`.
+
+## Restrictions
+
+
+## Example
+

```

Comment:
Rather than use straight JSON could you use the following includes:

```
{% include manifest_links.html viewers="UV, Mirador" manifest="manifest.json" %}

{% include jsonviewer.html src="manifest.json" config='data-line="15-27"' %}
```
The first one provides a link to a viewer (delete Mirador if it only works in the UV). 

The second one embeds a Javascript JSON viewer and can do line highlighting. It would be good to highlight the rendering section and maybe move that to the top of the manifest as its the most important part.

Reply:
You would also need to move the JSON below to its own file manifest.json

Reply:
I've fixed these issues now.